### DISCUSSION TITLE
Authentication với JWT, lưu token ở đâu là bảo mật nhất?

### DISCUSSION THREAD

**Discussion by giscus** [nvtaikma/Comment-blogs#7] [General]
# Authentication với JWT, lưu token ở đâu là bảo mật nhất?

http://localhost:5000/blog/authentication-voi-jwt-luu-token-o-dau-la-bao-mat-nhat-1710659545451

**Comment by nvtaikma** [comment]
adsdsad

**Comment by nvtaikma** [comment]
adsads
